spec/shoryuken/worker/inline_executor_spec.rb

```diff
@@ -46,4 +46,54 @@
       end
     end
   end
+
+  describe 'InlineExecutor with custom middleware' do
```

Comment:
@phstc 
When I run rspec with only this file(I mean spec/shoryuken/worker/inline_executor_spec.rb), nothing fail. However running rspec with all files, this file examples are failed all. Could u help me with this situation?